DatadogSessionReplay/Tests/Mocks/RUMContextMocks.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import TestUtilities
+
+@testable import DatadogSessionReplay
+
+extension RUMContext: AnyMockable, RandomMockable {
```

Comment:
probably needs `#if os(iOS)` as the rest of the mocks

Reply:
this has nothing to do with SR as per say, hence kept it outside. No issues so far.